app/views/movies/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1> <%= @movie.title %> Show Page </h1>
+
+<p> Created: <%= @movie.creation_year %> </p>
+<p> Genre: <%= @movie.genre %> </p>
+
+<p> Actors in Film </p>
+<% @sorted_actors.each do |actor| %>
+  <p> <%= actor.name %> </p>
+  <p> <%= actor.age %> </p>
+<% end %>
+
+<p>Average age of all actors: <%= @average_age %></p>
+
+<p> Add an Actor </p>
+
+<%= form_with url: "/movies/#{@movie.id}", method: :post, data: {turbo: false} do |form| %>
```

Comment:
While your controller setup is good, this path is a bit odd - granted, we have the first part correct (`"/movies/:id"`) but this also makes it look like we're creating a new movie. Maybe if we added `/actors` to our path, it would distinguish from just a single movie that we'd be creating?